app/views/words/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= provide(:title, "Create Word") %>
+<div class="text-center mt-4">
+    <h1>Create a new word</h1>
+</div>
+<div class="container">
+  <%= form_for(@word) do |f| %>
```

Comment:
please move the following to a partial file called `_form.html.erb` and replace it with `<%= render 'form' %>`